airbyte_cdk/sources/declarative/retrievers/simple_retriever.py

```diff
@@ -623,6 +619,7 @@ def _fetch_next_page(
                 stream_slice=stream_slice,
                 next_page_token=next_page_token,
             ),
+            # FIXME remove this implementation and have the log_formatter depend on the fact that the logger is debug or not
```

Comment:
I would like to remove the SimpleRetriever decorator eventually. We just need to have the log_formatter be optional be an optional field of the SimpleRetriever and we should be fine.

In theory, this seems good because it would allow for test reads configuration to apply to any retriever, not just the SimpleRetriever